components/monitoring/grafana/README.md

```diff
@@ -75,7 +75,7 @@ $ ./hack/setup-monitoring.sh grafana-datasource-secret $DATASOURCE_OPENSHIFT $PR
 ```
 
 Notes: 
-- The `grafana-oauth` service account is created by `components/monitoring/base/prometheus/configure-prometheus.yaml` along with a binding to the `cluster-monitoring-view` cluster role. 
+- The `grafana-oauth` service account is created by `components/monitoring/prometheus/base/configure-prometheus.yaml` along with a binding to the `cluster-monitoring-view` cluster role. 
```

Comment:
I think that this is the correct path
`components/monitoring/grafana/base/grafana-app.yaml`

Reply:
The ServiceAccount itself is created in `configure-prometheus.yaml`, while `grafana-app` seems to manage the application, configmaps, dashboard and environment for the whole grafana service

from `configure-prometheus.yaml`:
```
---
# The `grafana-oauth` service account is used by Grafana to fetch the metrics via the Auth proxy container of the Prometheus pods
---
apiVersion: v1
kind: ServiceAccount
metadata:
  annotations:
    openshift.io/description: used by Grafana to fetch the metrics via the Auth proxy container of the Prometheus pods
  namespace: appstudio-workload-monitoring
  name: grafana-oauth
---
```